CONTRIBUTING.md

```diff
@@ -45,3 +46,11 @@ If you need to regenerate the OpenAPI-generated, use the script:
 npm run clean
 npm run build
 ```
+
+## Release
+
+When you are ready to release, make sure the version in package.json has been bumped. Then create a [GitHub Release](https://github.com/sajari/sdk-node/releases).
```

Comment:
It might be nice to include a gh cli example, e.g.

```
gh release create v4.5.1 --generate-notes --draft
```

Note, the draft needs to be published, which can be done at the printed URL.